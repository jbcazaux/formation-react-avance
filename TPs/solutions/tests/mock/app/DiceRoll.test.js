import React from 'react'
import { render } from '@testing-library/react'
import DiceRoll from './DiceRoll'

jest.mock('./NiceDice', () => () => <div>NiceDice</div>)
jest.mock('components/button/Button', () => ({children}) => <div>{children}</div>)
describe('NiceDice', () => {
  it('renders DiceRoll', () => {
    const { container } = render(<DiceRoll />)

    expect(container).toMatchSnapshot()
  })
})
