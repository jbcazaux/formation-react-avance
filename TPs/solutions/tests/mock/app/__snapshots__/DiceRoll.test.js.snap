// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DiceRoll renders DiceRoll 1`] = `
<div>
  <div>
    <div>
      NiceDice
    </div>
    <div>
      Lancer le d√© !
    </div>
  </div>
</div>
`;
